### PR TITLE
added a redis insatance

### PR DESCRIPTION
**Pull request by gecBurton**
## Context

As a platform engineer I want an instance of Redis so that I can pass messages between the core-api and the workers.

## Changes proposed in this pull request

* elasticache instance added

## Guidance to review

* `make tf_apply`
* https://eu-west-2.console.aws.amazon.com/elasticache/home?region=eu-west-2#/redis/redbox-redbox-default-redis

## Relevant links

https://github.com/orgs/i-dot-ai/projects/10/views/1?pane=issue&itemId=59739503

## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [ ] I have tested any code added or changed
- [ ] I have run integration tests

### CODE REVIEW

File: infrastructure/aws/universal/variables.tf
Comment (ghannay10) [inline infrastructure/aws/universal/variables.tf:1]: Does the `tf_apply_universal` still work without this?

File: infrastructure/aws/main.tf
Comment (ghannay10) [inline infrastructure/aws/main.tf:1]: Can you rename this to be `elasticache.tf`?

**Review by ghannay10 — APPROVED** [review]
2 comments, but have approved

File: infrastructure/aws/universal/variables.tf
Comment (gecBurton) [reply infrastructure/aws/universal/variables.tf:1]: i havent checked... will do so now

Comment (gecBurton) [reply infrastructure/aws/universal/variables.tf:1]: no, fixed now

**Pull request merged**